components/common/Icon/inlineImages/AttachFile.tsx

```diff
@@ -0,0 +1,8 @@
+type PropsType = {
```

Comment:
The PropsType can be reused by all the icons in /inlineImages

Reply:
Fixed by moving props into interfaces file